boa/benches/bench_scripts/array_pop.js

```diff
@@ -1,24 +1,221 @@
-(function(){
-    let testArray = [83, 93, 27, 29, 2828, 234, 23, 56, 32, 56, 67, 77, 32,
-                     45, 93, 17, 28, 83, 62, 99, 36, 28, 93, 27, 29, 2828,
-                     234, 23, 56, 32, 56, 67, 77, 32, 45, 93, 17, 28, 83, 62,
-                     99, 36, 28, 93, 27, 29, 2828, 234, 23, 56, 32, 56, 67,
-                     77, 32, 45, 93, 17, 28, 83, 62, 99, 36, 28, 93, 27, 29,
-                     2828, 234, 23, 56, 32, 56, 67, 77, 32, 45, 93, 17, 28,
-                     83, 62, 99, 36, 28, 93, 27, 29, 2828, 234, 23, 56, 32,
-                     56, 67, 77, 32, 45, 93, 17, 28, 83, 62, 99, 36, 28, 93,
-                     27, 29, 2828, 234, 23, 56, 32, 56, 67, 77, 32, 45, 93,
-                     17, 28, 83, 62, 99, 36, 28, 93, 27, 29, 2828, 234, 23,
-                     56, 32, 56, 67, 77, 32, 45, 93, 17, 28, 83, 62, 99, 36,
-                     28, 93, 27, 29, 2828, 234, 23, 56, 32, 56, 67, 77, 32,
-                     45, 93, 17, 28, 83, 62, 99, 36, 28, 93, 27, 29, 2828, 234,
-                     23, 56, 32, 56, 67, 77, 32, 45, 93, 17, 28, 83, 62, 99,
-                     36, 28, 93, 27, 29, 2828, 234, 23, 56, 32, 56, 67, 77, 32,
-                     45, 93, 17, 28, 83, 62, 99, 36, 28];
+(function () {
+  let testArray = [
```

Comment:
I am not 100% sure of this array formatting, but I guess it's the standard JS formatting.